### ISSUE TITLE
!!! maya2glTF: failed at 17:07:06 !!!

### ISSUE THREAD

**Issue by 3dnoobieee** [iimachines/Maya2glTF#199]
Hello. I tried to export selected mesh but this error appeared !!! maya2glTF: failed at 17:07:06 !!!
What can I do to avoid this problem? Thank you in advance.

Maya 2020.2
Microsoft defender is off
Windows 11

**Comment by ziriax** [comment]
Can you paste the full error log?

**Comment by 3dnoobieee** [comment]
I hope this is the needed text. I found it script editor.

Remove animation from 'fluidShape2Cache1', or delete the node entirely, to resume caching.


# Warning: Cached Playback: Safe mode - cache disabled as unsupported nodes were found in the scene. See the Script Editor for details. # 
maya2glTF -sn "Steam" -of "C:/Users/krist/OneDrive/Рабочий стол/Steam/export/" -sf 1 -cpr "(C)2024 krist"  -sno -vno
maya2glTF: loading plugin...
// Warning: file: C:/Users/krist/OneDrive/Документы/maya/2020/scripts/maya2glTF_UI.mel line 217: Plug-in, "maya2glTF.mll", is already loaded.  Skipped. // 
maya2glTF: plugin loaded.
// Warning: line 1: Maya failed to provide the tangents of mesh 'polySurfaceShape1'!
Assign texture coordinates and/or cleanup your mesh and try again please // 
// Error: line 1: Couldn't write to 'C:/Users/krist/OneDrive/xD0xE0xE1xEExF7xE8xE9 xF1xF2xEExEB/Steam/export/Steam0.bin' // 
!!! maya2glTF: failed at 18:12:53 !!!
updateRendererUI;
The dynamics evaluator must disable the evaluation manager due to these unsupported nodes: fluidShape2 fluidEmitter1
    Disable the 'dynamics' evaluator, or remove the nodes, to resume using the evaluation manager.
# Warning: Cached Playback: Cache disabled as the evaluation manager has been disabled. See the Script Editor for details. # 
maya2glTF -sn "Steam" -of "C:/Users/krist/OneDrive/Рабочий стол/Steam/export/" -sf 1 -cpr "(C)2024 krist"  -acn "clip" -ast 1 -aet 300 -afr 30 -sno -vno
maya2glTF: loading plugin...
maya2glTF: plugin loaded.
// Error: line 1: maya2glTF [18:13:06] *** ERROR ***  [(kFailure): Object does not exist] in MeshIndices::MeshIndices (C:/dev/glTF/Maya2glTF/src/MeshIndices.cpp:23) // 
!!! maya2glTF: failed at 18:13:06 !!!

**Comment by ziriax** [comment]
Can you try to export to a folder not in onedrive?

**Comment by 3dnoobieee** [comment]
I tried, doesn't help

**Comment by ziriax** [comment]
Can you share you stage and the object you select, or is it private?

**Comment by 3dnoobieee** [comment]
Sure
![Screenshot_1](https://github.com/iimachines/Maya2glTF/assets/173929091/ca4769a0-2209-41e8-898e-bd206039a7a6)
![Screenshot_2](https://github.com/iimachines/Maya2glTF/assets/173929091/c9f0f53f-d218-4845-a68b-762f1e96b842)
![Screenshot_3](https://github.com/iimachines/Maya2glTF/assets/173929091/6d7d1093-fc1f-4e47-8667-d27d50b44fb8)

**Comment by 3dnoobieee** [comment]
I can send more screenshots if needed

**Comment by ziriax** [comment]
No I mean can you share the Maya scene?

**Comment by 3dnoobieee** [comment]
Yes sure. But this site doesn't support that type of files. Maybe I can Email you?
